pages/admin/post/CreatePost.js

```diff
@@ -0,0 +1,113 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import TextField from "@mui/material/TextField";
+import InputLabel from "@mui/material/InputLabel";
+import MenuItem from "@mui/material/MenuItem";
+import FormControl from "@mui/material/FormControl";
+import Select from "@mui/material/Select";
+import { useState, useEffect } from "react";
+import axiosClient from "../../../axiosClient";
+import {
+  Card,
+  CardContent,
+  CardActions,
+  Typography,
+  Button,
+} from "@mui/material";
+function CreatePost() {
+  const [title, setTitle] = useState("");
+  const [tags, setTags] = React.useState([]);
+  const [content, setContent] = useState("");
+  const [tempTag, setTempTag] = useState("");
+  const handleChangeTitle = (e) => {
+    e.preventDefault();
+    setTitle(e.target.value);
+  };
+  const handleChangeTag = (e) => {
+    e.preventDefault();
+    setTempTag(e.target.value);
+  };
+  const handleChangeContent = (e) => {
+    e.preventDefault();
+    setContent(e.target.value);
+  };
+  useEffect(() => {
+    async function FetchTag() {
+      const result = await axiosClient.get("/tags");
+      setTags(result.data);
+    }
+    FetchTag();
+  }, []);
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const body = JSON.stringify(content);
+
+    try {
+      const createPost = axiosClient.post("/posts", {
+        Title: title,
+        Content: body,
+        Tags: tempTag,
+      });
+    } catch (error) {
+      console.log(error);
+    }
+  };
```

Comment:
Is this component made for 'Admin create announcement' use case? If yes, I think you've missed some properties and mechanics. You need a User's data (i.e. Admin's data) that post this announcement, then you should link this to notification content-type (add new or something) and mark this post as an announcement. And you should consider the way users receive it.

Reply:
I created a post I took from Phuc Nguyen, so admin didn't create a post :)

Reply:
Ok sure, keep it for later, mark my words when you want to do it.